packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss

```diff
@@ -46,7 +45,7 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  height: calc(100vh - 6rem);
```

Comment:
@Madhu-mac changing the height to `100vh` breaks the UI on the desktop viewport. The correct values are:

- var(--desktop-workspace-window-height) for desktop
- var(--tablet-workspace-window-height) for tablet